### ISSUE TITLE
draw_arc always draws the shortest arc between start and end points, ignores order

### ISSUE THREAD

**Issue by JohanAR** [godotengine/godot#64685]
### Godot version

v4.0.alpha.custom_build [f8cfcf3e8]

### System information

Xubuntu 22.04

### Issue description

CanvasItem draw_arc appears to ignore the order of start and end points.

`draw_arc(Vector2(50, 50), 40, 0, PI/2, 32, Color.WHITE, 2, true)`

Will draw an arc clockwise from 0 to PI/2 as expected.

`draw_arc(Vector2(150, 50), 40, PI/2, 0, 32, Color.WHITE, 2, true)`

Will draw an arc counter clockwise from PI/2 to 0, which results in exactly the same arc as the first one. I would expect this to draw the "complementary" arc to the first one, i.e. a clockwise arc from PI/2 to 0

`draw_arc(Vector2(250, 50), 40, -3*PI/2, 0, 32, Color.WHITE, 2, true)`

It is necessary to use a negative angle to trick it into drawing the clockwise arc from PI/2 to 0

This behaviour is undocumented, and IMO impractical and slightly confusing

### Steps to reproduce

```
extends Node2D

func _draw():
	# Order of start and end points does not matter
	draw_arc(Vector2(50, 50), 40, 0, PI/2, 32, Color.WHITE, 2, true)
	# It draws exactly the same arc
	draw_arc(Vector2(150, 50), 40, PI/2, 0, 32, Color.WHITE, 2, true)
	# Have to use negative angles to trick it to draw the long arc
	draw_arc(Vector2(250, 50), 40, -3*PI/2, 0, 32, Color.WHITE, 2, true)

```

![image](https://user-images.githubusercontent.com/4918549/185786585-55bbd455-8773-42e1-8bd1-fbb9d3d893a2.png)

**Comment by JohanAR** [comment]
@Calinou is it certain that the current behaviour is intended, so the solution is to update the documentation? Personally I would think it more logical if the arc was always drawn clockwise, so changing the order of the arguments would invert which part of the circle is drawn, but maybe that's just me :)

If we're updating the documentation, adding 2*PI to one of start or end ought to also "invert" the arc. I wrote negative angles, because that's what I came up with while experimenting with this, but on second thought I just subtracted 2*PI from one argument.
